pkg/util/util.go

```diff
@@ -856,16 +862,48 @@ func Unzip(src, dest string) ([]string, error) {
 	}
 	defer r.Close()
 
-	for _, f := range r.File {
+	// change path separator to correct character for windows
+	if runtime.GOOS == "windows" {
+		pathToUnzip = strings.Replace(pathToUnzip, "/", string(os.PathSeparator), -1)
+	}
 
+	for _, f := range r.File {
 		// Store filename/path for returning and using later on
-		index := strings.Index(f.Name, "/")
+		index := strings.Index(f.Name, string(os.PathSeparator))
 		filename := f.Name[index+1:]
```

Comment:
you could probably use https://golang.org/pkg/path/filepath/#Base here instead of using getting the index of `/` then breaking the path.